### ISSUE TITLE
Abstract gets truncated

### ISSUE THREAD

**Issue by flaneuse** [outbreak-info/litcovid#8]
Due to special characters?  Not sure what's up.  https://outbreak.info/resources/pmid31614176

**Comment by marcodarko** [comment]
There is <sub> tags, whatever that means, that are cutting things off..

`<AbstractText>
Neuro-inflammation might impact on clinical manifestations and cognition function via changing the volumes of key brain structures such as the anterior cingulate cortex (ACC) in bipolar disorder (BD). In this study, we investigated the interrelations among interleukin (IL)-6 cytokine level, grey matter (GM) volume of the anterior cingulated cortex (ACC), and attention function among offspring of parents diagnosed with BD. The offspring were categorized as being either asymptomatic or symptomatic based on whether they manifested pre-defined sub-threshold mood symptoms. We found that the symptomatic offspring showed significantly higher serum levels of IL-6 than the asymptomatic offspring (F
<sub>(1, 59)</sub>
 = 67.65, p < 0.001). On the brain level, we obtained significant interactive effect of group and IL6 level on the ACC GM (P
<sub>FWE</sub>
 = 0.017). Specifically, the GM volume of the rostral ACC was negatively associated with the levels of IL-6 in the asymptomatic offspring (P
<sub>FWE</sub>
 = 0.021), but not the symptomatic offspring (P
<sub>FWE</sub>
 > 0.05). Mediation analyses revealed that the GM volume of the rostral ACC significantly mediated the negative association between the IL-6 levels and attention performance in the asymptomatic offspring (bootstrapping Confidence Interval (CI) = -6.0432 to -0.0731) but not the symptomatic offspring (bootstrapping CI = -0.3197 to 1.3423). Our data suggest that the asymptomatic and symptomatic bipolar offspring may exhibit different neurocognitive-inflammatory profiles, which could be further validated as viable biosignatures for BD risk and resilience.
</AbstractText>`
